Tests/js/views/ez-languageswitcherview.html

```diff
@@ -0,0 +1,90 @@
+<!doctype html>
+<html>
+<head>
+<title>eZ Language Switcher View tests</title>
+</head>
+<body>
+
+<div class="click-somewhere"></div>
+
+<div class="container"></div>
+
+<script type="text/x-handlebars-template" id="languageswitcherview-ez-template">
+    <div class="ez-selected-language">
+        {{#if otherTranslations}}
+            <a href="#" class="ez-dropdown-list-indicator" data-test="{{ location.id }}">
+                <span class="ez-language-flag flag-icon flag-icon-{{{ getFlagCode currentTranslation }}}"></span>
```

Comment:
here as well, the template can be simplified. Please remove any call to getFlagCode (it's not needed for the test) and this allows to remove the helper dependency below.